I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 282, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\potat\OneDrive\Desktop\Игры)\renpy-8.3.7-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\potat\OneDrive\Desktop\Игры)\renpy-8.3.7-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "C:\Users\potat\OneDrive\Desktop\Игры)\renpy-8.3.7-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\potat\OneDrive\Desktop\Игры)\renpy-8.3.7-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 282, in script
    python:
  File "C:\Users\potat\OneDrive\Desktop\Игры)\renpy-8.3.7-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
Sapphos_stories 1.0
Tue Jul 22 00:37:10 2025
